core/src/main/java/org/apache/carbondata/core/util/DataTypeUtil.java

```diff
@@ -292,7 +292,7 @@ public static Object getDataBasedOnDataType(String data, DataType actualDataType
             Date dateToStr = dateformatter.get().parse(data);
             return dateToStr.getTime() * 1000;
           } catch (ParseException e) {
-            LOGGER.error("Cannot convert" + data + " to Time/Long type value" + e.getMessage());
+            LOGGER.error("Cannot convert value to Time/Long type value" + e.getMessage());
```

Comment:
Concatenating 2 message  and logging will be better or adding two different log message will be better without newline(first message without newline)?? Because for each concatenation it will create new String object.
@ravipesala @gvramana Please comment!